lib/elixometer.ex

```diff
@@ -122,11 +122,12 @@ defmodule Elixometer do
     end
   end
 
-  defp build_timer_body(%Timer{} = timer_data) do
-    quote do
-      timed(unquote(timer_data.key), unquote(timer_data.units)) do
-        unquote(timer_data.body)
-      end
+  defp build_timer_body(%Timer{body: [do: block]} = timer) do
+    build_timer_body(%{timer | body: block})
```

Comment:
Unfortunately it is ambiguous. The issue is that keywords are list in AST:
```elixir
quote do
  [key: :value]
end
```
So the following would get different behaviour on 1.4 v 1.5:
```elixir
def foo() do
  [key: :value]
end
```